taxonomies/packaging_materials.txt

```diff
@@ -163,7 +163,7 @@ pt:PET opaco
 <en:Plastic
 en:HDPE - High-density polyethylene, High-density polyethylene
```

Comment:
why don't we have HDPE as the main word @teolemon or @stephanegigandet  (shan't we do it for every translation below ?)

Reply:
Because the main word is shown, and it's more pedagogical to see both